src/queue.js

```diff
@@ -219,4 +219,6 @@ class Queue extends EventEmitter {
   }
 }
 
+Queue.Job = null; // Make TypeScript compiler happy. Needed because Job is an abstract class.
```

Comment:
Is there no way around this. It seems rather silly to add a useless property to the public api just to keep a super-set language compiler happy.